CONTRIBUTING.md

```diff
@@ -72,6 +75,48 @@ If you do not find the suggestion, please open a new issue and clearly document
 
 Please provide your enhancement suggestions as an issue in the repository:
 
+### Dev Environments
+
+#### Manual Setup
+
+General guidance for setting up a local development environment in Linux are provided below.
+It is strongly recommended to perform this within a virtual environment such as [conda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) or [pyenv](https://akrabat.com/creating-virtual-environments-with-pyenv/):
+
+```bash
+# Checkout the repository
+git clone https://github.com/cytomining/pycytominer.git
+cd pycytominer
+# Install pycytominer in development mode along with associated tools
+bash .devcontainer/postCreateCommand.sh
+```
+
+#### Local DevContainer
+
+Instructions for setting up a local development environment using VSCode DevContainers:
+
+1. Install [VSCode](https://code.visualstudio.com/download)
+2. Install the [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension
+3. Open the repository in VSCode
+4. Click on the green "Reopen in Container" button in the lower left corner of the window
+5. Wait for the container to build and install the required dependencies
+
+#### Cloud Environment (Recommended)
```

Comment:
```suggestion
#### Cloud environment
```

We can add the `Recommended` flag to this header once we spend more time using this method :)

Feel free to add a more direct note re. recommended in the section below however. For example, might it be worth saying: "We recommend using the cloud development configuration over local dev containers"?

Also note the lower case header to keep consistent documentation

Reply:
Makes sense. Added a line recommending local/cloud dev containers over manual setup.